ci/test/cargo-test-miri.sh

```diff
@@ -19,4 +19,4 @@ set -euo pipefail
 export CARGO_TARGET_DIR="$PWD/miri-target"
 export MIRIFLAGS="-Zmiri-disable-isolation -Zmiri-strict-provenance"
 # exclude netwrok based tests, they mostly fail on epoll_wait
-cargo miri nextest run -j"$(nproc)" --no-fail-fast --workspace --exclude 'mz-adapter*' --exclude 'mz-environmentd*' --exclude 'mz-expr*' --exclude 'mz-compute-client*' --exclude 'mz-persist-client*' --exclude 'mz-ssh-util*' --exclude 'mz-rocksdb*'
+cargo miri nextest run -j"$(nproc)" --no-fail-fast --workspace --exclude 'mz-adapter*' --exclude 'mz-environmentd*' --exclude 'mz-expr*' --exclude 'mz-compute-client*' --exclude 'mz-persist-client*' --exclude 'mz-ssh-util*' --exclude 'mz-rocksdb*' --exclude 'mz-transform*'
```

Comment:
We are using a library (`stacker`) that grows the stack in IR traversals based on recursive calls in order to avoid stackoverflow, and for some reason one of my tests is trigggering a call from this library that is not supported b miri:

https://buildkite.com/materialize/tests/builds/53475#018774e8-3598-4fc1-a602-68324cc62b8d/141-1003

Reply:
If it's just the one test, you can also disable it explicitly by annotating it with `#[cfg_attr(miri, ignore)]`